### DISCUSSION TITLE
Make seperate function for holding keys

### DISCUSSION THREAD

**Discussion by phorgen** [cajhin/capsicain#96] [Ideas]
Hi, first I want to thank you for making such a great tool.
Second, I'd like to ask if it would be possible to make a function that, different to key(), holds down the key/combo as long as the key/combo that triggers it is held or makes any key toggleable. The primary usage I'd have for this is to fully restore keys rebound to other keys, because creating a new combo for every obscure combo some program seems really annoying. It would also just feel better to have keys fully restored.

**Comment by cajhin** [comment]
hi, I think I get some of what you ask, but not completely. Like "fully restore keys rebound to other keys, because creating a new combo for every obscure combo some program seems really annoying". 
Can you give me a real life example of how you want one key to work, and what the problem is?

**Reply by phorgen** [reply]
I want to suggest a function like this:
```
REWIRE LWIN MOD12
COMBO W [&... .... ....] > toggleKey(LWIN) 
```
The function should result in LWIN being held when the windows key + W key are pressed until the windows key + W key are pressed again, so that when you can (for example) press the E key to open file explorer, press D to show desktop or use any other function that uses LWIN. I used LWIN in this example because it is the only key I have rewired, but this is also the reason why I have yet to rebind the ALT key, because many applications use some obscure shortcuts with ALT, and I don't want to add a COMBO for all of them for every single program.
